.devcontainer/devcontainer.json

```diff
@@ -57,23 +58,37 @@
 				// "GitHub.copilot@prerelease",
 				// "GitHub.copilot-chat@prerelease"
 				"GitHub.copilot",
-				"GitHub.copilot-chat"
-			]
+				"GitHub.copilot-chat",
+				"specstory.specstory-vscode"
+			],
+			"mcp": {
+				"servers": {
+					"playwright": {
+						"command": "npx",
+						"args": [
+							"-y",
+							"@microsoft/mcp-server-playwright"
+						]
+					}
+				}
+			}
```

Comment:
Do we need `playwright` here if it's already in the `mcp.json`?

Reply:
Removed from devcontainer and kept only in mcp.json